### ISSUE TITLE
trap range too low?

### ISSUE THREAD

**Issue by Chulain** [ccshiro/cc-buglist#782]
https://www.youtube.com/watch?v=-F5bM3v09cA

trap range seems to be a bit too low?

retail: 
https://youtu.be/o-S9sDhN5IY?t=277

https://youtu.be/Q6SP7TqdaOQ?t=44
(very clear here, it's pre tbc tho)

**Comment by darkcranio** [comment]
Also there is a delay in PVP to traps, so if lucky enough, we can pass it.

**Comment by Stoney21** [comment]
Delay on traps is blizzlike.

**Comment by YourNewfriend** [comment]
https://youtu.be/nXZTIuJ_gHc?t=1m56s

**Comment by Stoney21** [comment]
Yeah, he put his trap and rogue went right through it. It should take around 2 seconds befor activation.

**Comment by ccshiro** [comment]
@pretbc video: Slow it down to 0.25 speed, notice that scatter shot makes him go much closer than he actually gets stunned at. Keep in mind that spell delay is a thing (as is lag). My guess is that the trap triggered when he went really close in scatter shot.

@tbc video. Same thing here, he runs past the trap fairly close to it. Spell delay in effect, he's now further away than the closest point he crossed it at.

You can indeed try this at our server and observer the same effect (just did it with nim), where you run past the trap and you get stunned further away than it would trigger normally.

**Comment by YourNewfriend** [comment]
https://youtu.be/kqc7G3f0kro?t=1m3s

**Comment by justinbeaver** [comment]
I think we should open this report for some time.

---

Advice from old hunter guide:

<i>"stand on the trap to force melee opponents to either dance around it or get frozen."
https://web.archive.org/web/20080224062154/http://forums.worldofwarcraft.com/thread.html?topicId=12884227</i>

Could be melee range (5 yards) is the <b>radius</b> where enemy can triggered a trap?
(<i>"defines 'Melee Range' as 5 Yards or closer" http://wowwiki.wikia.com/wiki/Melee?oldid=1433058</i>)

If hunter will stay right in the trap, anyone who hit him in the melee will get frozen? But could be different for different races because of the hitbox.

---

<b>World #1 2v2 Druid/Hunter</b>
Warlock is sunned so he can't run thru trap and then triggered it in the other spot due spell delay/lag.
![stunned_lock_0](https://cloud.githubusercontent.com/assets/13700367/9544574/95cd8e9a-4d8a-11e5-8a5c-f093ca59bf39.png)
4-5 yards?
![stunned_lock_1](https://cloud.githubusercontent.com/assets/13700367/9544575/97b10214-4d8a-11e5-9481-ade555303319.png)
Trap is triggered on warlock not on felhunter, you can tell it by center of trap. (Trap center is always on enemy that triggered it, not on initial trap spot). And warlock is still stunned.
![stunned_lock_2](https://cloud.githubusercontent.com/assets/13700367/9544576/99712c00-4d8a-11e5-9fd6-bf1ff53dd488.png)
<i>https://youtu.be/Mh6PN9VxEDY?t=747</i>
WCM link: <i>http://www.warcraftmovies.com/movieview.php?id=58495</i>

---

<b>Swarm 3 - BG9 5v5</b>
<b>6:22</b>
You can see everything yourself
![swarm3 wmv_snapshot_06 27_ 2015 08 17_06 05 16](https://cloud.githubusercontent.com/assets/13700367/9544933/78b9d572-4d8d-11e5-8d79-efefbe16f805.jpg)
Is it 5 yards?
![swarm3 wmv_snapshot_06 27_ 2015 08 17_06 06 18](https://cloud.githubusercontent.com/assets/13700367/9544934/7ba37a36-4d8d-11e5-85d1-1281c00068ea.jpg)
![swarm3 wmv_snapshot_06 28_ 2015 08 28_13 58 03](https://cloud.githubusercontent.com/assets/13700367/9544937/7d4c5a92-4d8d-11e5-8a5c-46f8c0863188.jpg)
Don't want to spam much with screenshots, so only one for each trap
<b>09:40</b>
Triggered (You can better see in the movie)
![swarm3 wmv_snapshot_09 42_ 2015 08 28_14 36 39](https://cloud.githubusercontent.com/assets/13700367/9546271/1b3b84ec-4d9a-11e5-92be-2301d8fde6a2.jpg)
<b>15:33</b>
Triggered
![swarm3 wmv_snapshot_15 33_ 2015 08 28_14 42 49](https://cloud.githubusercontent.com/assets/13700367/9545467/1d958d98-4d93-11e5-929e-a1c703e82339.jpg)
SK download: <i>http://www.sk-gaming.com/file/6229-Swarm_3__BG9_5v5</i>
WCM link: <i>http://www.warcraftmovies.com/movieview.php?id=62639#mirrors</i>

---

Now screenshot from ts movie
![qwegfd](https://cloud.githubusercontent.com/assets/13700367/9546514/f0a6b678-4d9b-11e5-9b9d-62d3ee34c8cc.png)
Pretty close i think.

Difference between my movies and posted earlier in this report that it's not a scatter-trap combo, so enemy didn't run over the trap and triggered it later with delay/lag.
But delay/lag can be the answer to range of trap. "<i>shiro: Keep in mind that spell delay is a thing (as is lag). My guess is that the trap triggered when he went really close in scatter shot.</i>"
That is explains why you sometimes can see in different movies that trap triggered on slightly different range from trap to enemy.
For example: enemy is moving towards the trap and it triggered at 5 yards (not sure how many yards should be) but due to delay/lag he make couple more steps and then trap debuff applies.

Pretty much how it looks like when enemy running out from trap
<b>2400++ all dps 2vs2/3vs3: Hunter Jubiter season 3 of Korea</b>
<b>5:34</b>
![untitled1](https://cloud.githubusercontent.com/assets/13700367/9546980/94d9fbd0-4d9f-11e5-8ed4-52a772d712c3.png)
![untitled2](https://cloud.githubusercontent.com/assets/13700367/9546982/98965926-4d9f-11e5-8d7b-c3e26021b533.png)
![untitled3](https://cloud.githubusercontent.com/assets/13700367/9546985/9ca6a53e-4d9f-11e5-942f-cb27f0484ce5.png)
![untitled4](https://cloud.githubusercontent.com/assets/13700367/9546986/9ffb5d60-4d9f-11e5-9499-d4ee2e525582.png)
WCM link: <i>http://www.warcraftmovies.com/movieview.php?id=82133</i>

---

And some more screenshots with scatter-trap combo form <b>Jacktrapper III 2300 rogue+hunter</b> <i>http://www.warcraftmovies.com/movieview.php?id=74611</i>
![still trapper 1](https://cloud.githubusercontent.com/assets/13700367/9547113/a63bb8a4-4da0-11e5-9688-bb2d2f187707.png)
![still trapper 2](https://cloud.githubusercontent.com/assets/13700367/9547116/a9079724-4da0-11e5-95fe-f57a67b7a65a.png)
![trapper warr1](https://cloud.githubusercontent.com/assets/13700367/9547118/abffca00-4da0-11e5-9dd6-72b4a56b326b.png)
![trapper warr2](https://cloud.githubusercontent.com/assets/13700367/9547119/ade971ae-4da0-11e5-81b2-520054fc603c.png)
![trapper1](https://cloud.githubusercontent.com/assets/13700367/9547121/afb743f8-4da0-11e5-858a-848391e46d0b.png)
![trapper2](https://cloud.githubusercontent.com/assets/13700367/9547125/b1be1eec-4da0-11e5-8362-bbe19e992dac.png)

---

And don't forget that in videos where hunter is kiting someone and placing a trap, arming time will affect where trap will be triggered in some way.

@darkcranio "Also there is a delay in PVP to traps, so if lucky enough, we can pass it."
@Stoney21 "Delay on traps is blizzlike."
"Yeah, he put his trap and rogue went right through it. It should take around 2 seconds befor activation."

It is arming time which start counting after hunter placed a trap not when enemy reached it, and I'm pretty sure it should be 2.0 sec, when in reality right now is closer to 3 :( https://github.com/ccshiro/cc-buglist/issues/904

**Comment by justinbeaver** [comment]
Right now on CC
Trap is placed right in the hunter

<b>Not in melee range</b> You are too far away
![1_toofaraway](https://cloud.githubusercontent.com/assets/13700367/9552630/4b88a59e-4dc2-11e5-8c83-0f9eebc7f190.jpg)
<b>In melee range</b> Small step closer
![2_melerange](https://cloud.githubusercontent.com/assets/13700367/9552633/4f15adf6-4dc2-11e5-96de-f8ebf6a64b81.jpg)
Closest you can stay untill freeze
![10](https://cloud.githubusercontent.com/assets/13700367/9552635/51438652-4dc2-11e5-90d4-43ed3f10bfe2.jpg)
<b>Freezed</b> One small step closer
![11](https://cloud.githubusercontent.com/assets/13700367/9552636/5348d038-4dc2-11e5-9061-752edf21e9a7.jpg)

---

In bear form
Closest you can stay
![wowscrnshot_082815_195356](https://cloud.githubusercontent.com/assets/13700367/9552798/6b1cf7a6-4dc3-11e5-93da-d3d2a58e6e3a.jpg)
Freezed
![wowscrnshot_082815_195404](https://cloud.githubusercontent.com/assets/13700367/9552799/6d7f8fb8-4dc3-11e5-9b15-7d96960d128e.jpg)

---

**Comment by Chulain** [comment]
yeah this definitely feels wrong, I was going to research this but I've been postponing it forever

**Comment by justinbeaver** [comment]
@Chulain haha same
I did couple screenshots long time ago, but to make a proper report I decided only today after I saw this topic.

**Comment by justinbeaver** [comment]
<b>Arena_2v2_Rogue-Priest.avi</b>
<b>28:13</b>
1.
Placed a trap, arming time
![arena_2v2_rogue-priest avi_snapshot_28 13_ 2015 08 29_17 57 19](https://cloud.githubusercontent.com/assets/13700367/9562824/3690f09c-4e80-11e5-93e6-a53887f98c02.jpg)
Still arming time
![arena_2v2_rogue-priest avi_snapshot_28 15_ 2015 08 29_17 57 41](https://cloud.githubusercontent.com/assets/13700367/9562825/38df9312-4e80-11e5-9e57-864955bcb12c.jpg)
Triggered here? 4-5 yards?
![arena_2v2_rogue-priest avi_snapshot_28 15_ 2015 08 29_17 57 51](https://cloud.githubusercontent.com/assets/13700367/9562828/3a8051ca-4e80-11e5-9b3a-d845a7272fdc.jpg)
Moving out
![arena_2v2_rogue-priest avi_snapshot_28 15_ 2015 08 29_17 57 57](https://cloud.githubusercontent.com/assets/13700367/9562830/3bdc6964-4e80-11e5-8978-87e947d24f48.jpg)
Debuff applied
![arena_2v2_rogue-priest avi_snapshot_28 16_ 2015 08 29_17 58 06](https://cloud.githubusercontent.com/assets/13700367/9562832/3d810798-4e80-11e5-9f93-c285f0c58d98.jpg)

---

2.
Sometime later
Hunter placed a trap, arming time
![arena_2v2_rogue-priest avi_snapshot_29 42_ 2015 08 29_18 04 00](https://cloud.githubusercontent.com/assets/13700367/9562858/ea9469fc-4e80-11e5-9bbd-f10256e3a136.jpg)
Still arming time
![arena_2v2_rogue-priest avi_snapshot_29 42_ 2015 08 29_18 04 11](https://cloud.githubusercontent.com/assets/13700367/9562860/ec6d4280-4e80-11e5-9c18-054da1b54d46.jpg)
![arena_2v2_rogue-priest avi_snapshot_29 42_ 2015 08 29_18 04 16](https://cloud.githubusercontent.com/assets/13700367/9562862/edd5208e-4e80-11e5-8cb1-c68296f2e167.jpg)
![arena_2v2_rogue-priest avi_snapshot_29 43_ 2015 08 29_18 04 23](https://cloud.githubusercontent.com/assets/13700367/9562864/eefea9a8-4e80-11e5-817d-7a5fa5a9905b.jpg)
![arena_2v2_rogue-priest avi_snapshot_29 43_ 2015 08 29_18 04 28](https://cloud.githubusercontent.com/assets/13700367/9562865/f0509168-4e80-11e5-9f52-dbee6c9eb08b.jpg)
Triggered here? 4-5 yards?
![arena_2v2_rogue-priest avi_snapshot_29 44_ 2015 08 29_18 04 34](https://cloud.githubusercontent.com/assets/13700367/9562866/f2f3ea8c-4e80-11e5-99e7-e5915dbbeeb9.jpg)
![arena_2v2_rogue-priest avi_snapshot_29 45_ 2015 08 29_18 04 43](https://cloud.githubusercontent.com/assets/13700367/9562868/f4703636-4e80-11e5-8def-d3d40453977f.jpg)
WCM link: <i>http://www.warcraftmovies.com/movieview.php?id=68683</i>
Gamefront download:<i>http://www.gamefront.com/files/Arena+2v2+Rogue+Priest/;9899698;/fileinfo.html</i>

**Comment by justinbeaver** [comment]
What I've noticed from movies that you got instantly frozen after you did a hit to hunter

**Comment by Chulain** [comment]
http://www.warcraftmovies.com/movieview.php?id=81717
36:25

how could this trap go off with trap delay? it defnitely had more range than on corecraft..
